Code/Framework/AzFramework/AzFramework/Viewport/CameraInput.h

```diff
@@ -337,6 +337,7 @@ namespace AzFramework
     private:
         PanAxesFn m_panAxesFn; //!< Builder for the particular pan axes (provided in the constructor).
         InputChannelId m_panChannelId; //!< Input channel to begin the pan camera input.
+        ClickDetector m_clickDetector; //!< Used to determine when a sufficient motion delta has occurred to begin the input.
```

Comment:
The doxygen comment here (and line 494) is slightly confusing to me. The description of m_clickDetector doesn't mention detecting clicks at all. To me it seems like a more accurate description would be something like "Used to determine when a sufficient motion delta has occurred, following a click, to begin the input."

Reply:
That's a good shout, I can update :+1: